test/e2e/e2e_tests_suite_test.go

```diff
@@ -34,13 +34,13 @@ var testEnv *envtest.Environment
 
 // Define utility constants for object names and testing timeouts/durations and intervals.
 const (
-	testNamespace = "openshift-sriov-network-operator"
-
 	timeout  = time.Second * 30
 	duration = time.Second * 300
 	interval = time.Second * 1
 )
 
+var testNamespace string = os.Getenv("NAMESPACE")
```

Comment:
Should we panic in the test if NAMESPACE is somehow not defined (i.e os.Gentenv() returns an empty string) ?

Reply:
Done